impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/implementation/builtin/metadata/BuiltinMetadataBeanTest.java

```diff
@@ -14,8 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.implementation.builtin.metadata;
+package org.jboss.cdi.tck.tests.full.decorators.implementation.builtin.metadata;
```

Comment:
The test doesn't seem to be strictly about decorators. IMHO, the addition here should be just `full`, not `full.decorators`.

Reply:
This comment actually applies to all files changed in this PR. A lot of the changed tests are not only about decorators, and those that specifically test decorators already have `decorator` in the package name, so with this change, the package name of those tests will contain the word `decorator` twice.